Makefile

```diff
@@ -45,6 +45,9 @@ help:
 	@echo "  make stop-postgres				-- stops the local postgres cluster"
 	@echo "  make create-postgres			-- create the local postgres cluster (only works on ubuntu 20.04)"
 	@echo "  make local-a4					-- patch to use local a4 (needs to have path ../adhocracy4)"
+	@echo "  make celery-worker-start		-- starts the celery worker in the background
```

Comment:
it is running celery in stdout (foreground) not in the background.
Also to be consistent with the above postgres make command, I'd put `start/stop/status` as a prefix rather than a suffix.

Reply:
@hklarner I think @m4ra was referring to the explanation of the command, it still says `-- starts the celery worker in the background` which is wrong. Changing `start-postgres` to `postgres-start` is fine with me, both ways work

Reply:
done